### PR TITLE
feat: CU-1ht1y4b slider

### PR DESCRIPTION
**Pull request by snqb**
This is needed in card https://app.clickup.com/t/1ht1y4b

### DEMO:

https://user-images.githubusercontent.com/19791699/148215658-7c56c615-0406-4788-95a5-8b9fae9f3775.mp4

### CODE REVIEW

**Comment by giardiv** [comment]
Task linked: [CU-1ht1y4b Create deployment form](https://app.clickup.com/t/1ht1y4b)

File: src/components/slider/slider.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import RcSlider from 'rc-slider';
+import { FlexProps, Flex } from 'rebass';
+import { useTheme } from 'emotion-theming';
+import { ITheme } from '../../theme/types';
+import Labeling from '../typography/labeling';
+import Value from '../typography/value';
+
+export interface Props extends Omit<FlexProps, 'css' | 'onChange'> {
```
Comment (guceda) [inline src/components/slider/slider.tsx:9]: Most of the components have the props named `componentNameProps`. Would you consider updating the name to keep consistency? You could also get rid of the renaming in the `index` file.

**Review by guceda — APPROVED** [review]
Added a non-blocking tiny suggestion. The rest looks great!

File: src/components/slider/slider.tsx (hunk 1 above)
Comment (snqb) [reply src/components/slider/slider.tsx:9]: Can't agree on this one: naming this as `SliderProps` would pollute the export scope of the Quartz, causing cases when the IDE wants to import stuff from `quartz/dist/slider/slider` :(

**Review by giardiv — APPROVED** [review]

**Pull request merged**